src/golang/cmd/server/request/parameters.go

```diff
@@ -15,23 +13,15 @@ const (
 // Extracts the parameters dictionary from the request, which maps
 // parameter name to its json-serialized string value.
 func ExtractParamsfromRequest(r *http.Request) (map[string]string, error) {
-	paramsInBytes, err := ExtractHttpPayload(
-		r.Header.Get(routes.ContentTypeHeader),
-		parametersKey,
-		false,
-		r,
-	)
-	if err != nil {
-		return nil, err
-	}
```

Comment:
issue 2: for some reason ContentTypeHeader is not set when we post a request without a  `files` parameter. We can just grab the parameter from the body of the request.

Reply:
So parameters were in both header and body?